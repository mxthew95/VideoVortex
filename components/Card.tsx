import React, { FC } from 'react'
import {
    StyleSheet,
    View,
    Text
} from 'react-native'

const Card: FC = (props) => {
    return (
        <View style={{ ...styles.card, ...props.style }}>{props.children}</View>
    );
}

const styles = StyleSheet.create({
    card: {
        shadowColor: 'black',
        shadowOffset: { width: 0, height: 2 },
        shadowRadius: 6,
        shadowOpacity: 0.26,
        elevation: 3,
        backgroundColor: '#ffffff',
        padding: 10,
        borderRadius: 10
    }
})

export default Card;