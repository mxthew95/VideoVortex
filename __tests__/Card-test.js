import React from 'react'
import {create} from 'react-test-renderer'
import Card from '../components/Card'

const tree = create(<Card/>);

test('snapshot', ()=>{
    expect(tree).toMatchSnapshot();
});