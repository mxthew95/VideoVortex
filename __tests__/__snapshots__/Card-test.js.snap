// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "borderRadius": 10,
      "elevation": 3,
      "padding": 10,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.26,
      "shadowRadius": 6,
    }
  }
/>
`;
